src/TestFramework/TestFramework/TestFramework.csproj

```diff
@@ -25,6 +29,15 @@
     <PackageReference Include="Polyfill" PrivateAssets="all" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <IsAotCompatible Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))">$(EnableIotAnalyzers)</IsAotCompatible>
```

Comment:
This is already set (but forcing `true`) on line 16

Reply:
deleted just few seconds before. :)